.travis.yml

```diff
@@ -2,27 +2,32 @@ language: php
 
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.composer/cache/files
```

Comment:
caching dependencies means everything can be loaded from cache, which is significantly faster than downloading archives from GitHub each job.